apps/financial-aid/web-osk/src/components/Files/Files.tsx

```diff
@@ -25,8 +25,22 @@ const Files = ({ header, uploadFiles, fileKey }: Props) => {
     onRetry,
   } = useFileUpload(uploadFiles)
 
+  const stringifyFile = (file: UploadFile) => {
+    return {
+      key: file.key,
+      name: file.name,
+      size: file.size,
+      status: file.status,
+      percent: file?.percent,
+    }
+  }
```

Comment:
Im not sure, it is not used anywhere else, and maybe nice to have it there to see what it does and if you need to add a attribute or some? 
I think we should move it to utils if it is used somewhere else, what do think about that? 🤓

Reply:
Your choice, what is nice about having it in another file is that the name of the function says what we are doing, so you don't need to know about the implementation to use it. 
It also becomes more useable and less hidden since it is in a shared file and not in a component. Also it becomes more testable since the logic is isolated.